components/definition/cellRules/NetPyNECellRule.js

```diff
@@ -48,7 +47,27 @@ export default class NetPyNECellRule extends React.Component {
   componentWillReceiveProps(nextProps) {
     this.setState({ currentName: nextProps.name});
   }
-
+  
+  closeImportCell = () => {
+    this.setState({ importCellOpen: false });
+  };
+  
+  openImportCell = () => {
+    this.setState({ importCellOpen: true });
+  };
+  
+  postProcessMenuItems(pythonData, selected) {
+    return pythonData.map((name) => (
+      <MenuItem
+        key={name}
+        insetChildren={true}
+        checked={selected.indexOf(name) > -1}
+        value={name}
+        primaryText={name}
+      />
+    ));
+  };
+  
   render() {
     var that = this;
```

Comment:
I know this is not your code but I don't think we need `var that = this;` [In here](https://github.com/MetaCell/geppetto-netpyne/pull/15/files#diff-ba25582fe553b114216bf9ec24461e0aR118), you can simply use this